packages/tasit-sdk/src/scripts/checkDecentralandContractsData.js

```diff
@@ -0,0 +1,83 @@
+//
+// Script used to check if the target chain has the Decentraland app pre-conditions
+//
+// Note: This script is using mocha for convenince but isn't a test suite to be runned by `test` script.
+// Use: npx mocha src/scripts/checkDecentralandContractsData.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts
```

Comment:
Worth exposing this with an npm script?

Reply:
Yes, for sure. But about run it within the test suite script? I'm not sure.

Reply:
I agree, we wouldn't want it to run along with the test suite script

Reply:
So we'll still expose this script as a non-test npm script, though?

Reply:
Yes, done.

Reply:
Thanks! Resolving this.